sdcm/nemesis.py

```diff
@@ -2335,7 +2354,10 @@ def disrupt_delete_by_rows_range(self):
         else:
             partitions_for_exclude = self.delete_by_range_using_timestamp(ks_cf)
 
-        self.delete_range_in_few_partitions(ks_cf, partitions_for_exclude)
+        try:
+            self.delete_range_in_few_partitions(ks_cf, partitions_for_exclude)
+        except UnsupportedNemesis:
```

Comment:
Why are you catching this here ?

Why not just let it bubble up ?

Just for a warning? Which you already have a print for it when this raised ?

Reply:
@fruch , this is a bug fix. please see https://github.com/scylladb/scylla-cluster-tests/issues/6576

Reply:
Sorry this doesn't explain what is the solution here

And no comments here to explain it either


I don't under why it's o.k. to ignore and keep going, if we need to delete partions and there's no partions to delete why do we carry on with the nemesis

Reply:
@fruch, i think, following some later implementation, this could be fixed in a different way now.
I rolled-back the following:
```
-        try:
-            self.delete_range_in_few_partitions(ks_cf, partitions_for_exclude)
-        except UnsupportedNemesis:
-            self.log.warning('Could not find enough partitions for deletion. Nemesis is unable to run all its steps.')
+        self.delete_range_in_few_partitions(ks_cf, partitions_for_exclude)
```
Where `delete_range_in_few_partitions` has now a different exception:
```
-            raise UnsupportedNemesis('Not found partitions for delete. Nemesis can not be run')
+            raise Exception("No partitions for deletion found. Cannot execute a range deletion.")
```